data/presets/fields/defibrillator_location.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We usually use subdirectories for tags with colons. Could you move/rename this file to `fields/defibrillator/location.json`?